packages/webapp/src/components/Task/PureTaskTypeSelection/PureTaskTypeSelection.jsx

```diff
@@ -127,13 +127,15 @@ export const PureTaskTypeSelection = ({
             ?.filter(({ farm_id, task_translation_key }) => {
               const supportedTaskTypes = getSupportedTaskTypesSet(isAdmin);
               // If trying to make a task through the crop management plan 'Add Task' link -- exclude animal tasks from selection for now
-              const isNotAnimalTaskWhileCreatingCropTask = !(
-                ANIMAL_TASKS.includes(task_translation_key) && isMakingCropTask
-              );
+              const isNotAnimalTaskWhileCreatingCropTask =
+                isMakingCropTask && !ANIMAL_TASKS.includes(task_translation_key);
+              // If trying to make a task through the animal inventory 'Create a task' action -- exclude crop tasks from selection
+              const isAnimalTaskWhileCreatingAnimalTask =
+                isMakingAnimalTask && ANIMAL_TASKS.includes(task_translation_key);
               const shouldDisplayTaskType =
                 farm_id === null &&
                 supportedTaskTypes.has(task_translation_key) &&
-                isNotAnimalTaskWhileCreatingCropTask;
+                (isNotAnimalTaskWhileCreatingCropTask || isAnimalTaskWhileCreatingAnimalTask);
```

Comment:
Something seems missing, default task types are not displayed when creating a task from `/tasks` 🤔

Reply:
I got really confused last night trying to make this make sense 😂 I think I fixed it and cleaned the code up a bit too, let me know what you think

Reply:
It’s much easier to understand now, and it seems to be working correctly!